plutus-chain-index/e2e-tests/README.md

```diff
@@ -0,0 +1,15 @@
+# Chain-index end-to-end tests
+
+## Sanity API tests
+
+### Pre-requisites:
+- You have a testnet instance of chain-index running somewhere
+- Chain-index is synced to at least `min_slot` in the environment file `sanity-tests/testnet-env.json`
+- Update URL in environment file if needed
+
+### Run tests:
+- `nvm use`
```

Comment:
BTW, I think this is optional for users of `nix-shell` as `node` is already available (currently v14.17.4).

Reply:
Sure. I could remove this cmd and nvmrc file and we always assume run in nix-shell..?

Reply:
Yeah exactly! However, you can leave this for those people not using `nix-shell`, and say that it's unnecessary for those using `nix-shell`